sale_order_line_input/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2018 Tecnativa - Carlos Dauden
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    'name': 'Sale Order Line Input',
+    'summary': 'Adds menu option to work directly with sale lines and create'
```

Comment:
Don't make this too large, as it's rendered on the kanban view. You can put: "Search, create or modify directly sale order lines"